.vscode/launch.json

```diff
@@ -0,0 +1,11 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Attach to .NET Functions",
+            "type": "coreclr",
+            "request": "attach",
+            "processId": "${command:azureFunctions.pickProcess}"
+        }
+    ]
+}
```

Comment:
Git gets really upset if we don't add newlines at the end of files. This isn't a serious issue, but we still try to do it to avoid the big red no icon haha. 

Can you please add new lines at the end of your files? :)